.github/workflows/pytest.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           predicate-quantifier: "every"
           filters: |
             code:
+              - '!docker/*'
```

Comment:
I suppose that, while changes in `docker/*` can certainly break the builds, mostly CI won't catch that

Reply:
We attempt to build the Docker container, and we have shellcheck linting the scripts, so I think we will catch the real dumdum mistakes (which are checked outside of the integration tests) but yeah right now regardless of what we do here the only real test is attempting to do a deployment.